### PR TITLE
feat(batchenforce): added batchEnforce

### LINKED ISSUE #321: BatchEnforce
I don't see the BatchEnforce API(https://casbin.org/docs/ja/data-permissions) for the coreEnforcer. Is it not available yet?

### PR DESCRIPTION
**Pull request by Shivansh-yadav13**
Added BatchEnforce

fix #321

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Zxilly @Gabriel-403 @nodece

**Review by Zxilly — APPROVED** [review]
lgtm

File: src/coreEnforcer.ts [hunk 1]
```diff
@@ -588,4 +588,18 @@ export class CoreEnforcer {
...
+   * @param rvals the request need to be mediated, usually an array
+   *              of array of strings, can be class instances if ABAC is used.
+   * @returns whether to allow the requests.
+   */
+  public async batchEnforce(rvals: any[]): Promise<boolean[]> {
+    const results: boolean[] = [];
+    for (const rval of rvals) {
+      results.push(await this.enforce(...rval));
+    }
+    return results;
```
Comment (nodece) [inline src/coreEnforcer.ts:603]: Use `Promise.all` should be better, so like:
```js
return await Promise.all(rvals.map(n => this.enforce(...n));
```

**Review by nodece — CHANGES_REQUESTED** [review]

File: src/coreEnforcer.ts (hunk 1 above)
Comment (Shivansh-yadav13) [reply src/coreEnforcer.ts]: 👍

**Pull request merged**